Formula/ccls.rb

```diff
@@ -0,0 +1,37 @@
+class Ccls < Formula
+  desc "C/C++ language server"
+  homepage "https://github.com/MaskRay/ccls"
+  url "https://github.com/MaskRay/ccls.git", :tag => "0.20180513", :revision => "19d0aad2ca9663b5633b575efe41803005b21d38"
+  head "https://github.com/MaskRay/ccls.git"
+
+  option "with-build-debug", "Configures ccls to be built in debug mode"
```

Comment:
No options: build without debug and with Homebrew's clang.

Reply:
Default means "build without debug and with Homebrew's clang".

Reply:
Please remove the options, so all users benefit from prebuilt binaries.